packages/website/docs/_components_pages/main/Table/TableCell.mdx

```diff
@@ -3,6 +3,22 @@ slug: ../../TableCell
 sidebar_class_name: newComponentBadge expComponentBadge
 ---
 
+import HAlign from "../../../_samples/main/Table/HAlign/HAlign.md";
+
 <%COMPONENT_OVERVIEW%>
 
 <%COMPONENT_METADATA%>
+
+### Horizontal cell alignment
+
+Control the horizontal alignment of cells by utilizing the `horizontalAlign` property.
+
+Please note that the behaviour of `horizontalAlign` depends on where you set it:
+1. `horizontalAlign` is set on `TableHeaderCell` level<br />
```

Comment:
Could you use a bulleted list? This would be more suitable IMO , since this is not a logical order or procedure.